deploying-as-a-pod.md

```diff
@@ -1,8 +1,6 @@
-## Deploying as a pod
+# Deploying as a pod
 
-See this page for all [Kubecost install options](http://docs.kubecost.com/install).
```

Comment:
How do you feel about keeping a pointer to here? https://www.opencost.io/docs/install

This allows us to always keep a single source of truth in terms of full set of install options.

Reply:
For sure, this doc needs a little work. Will add this to the top.